IBAnimatableTests/ColorTypeTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ColorTypeTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/26/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
```

Comment:
@tbaranes I tried changing this to a normal import but it clashes with the mock protocols that I only want to give a `fileprivate` scope.